src/Controls/src/Core/Platform/Windows/Extensions/TextBlockExtensions.cs

```diff
@@ -99,6 +64,46 @@ public static void UpdateDetectReadingOrderFromContent(this TextBlock platformCo
 				platformControl.SetTextReadingOrder(label.OnThisPlatform().GetDetectReadingOrderFromContent());
 		}
 
+		internal static void SetLineBreakMode(this TextBlock textBlock, LineBreakMode lineBreakMode, int? maxLines = null)
```

Comment:
Do we need to add this method and mix the max lines and line break? I know the other platforms need it, but Widows is a bit better in this area?

Reply:
Yeah, both properties are related so like in other platforms we have a method to manage everything in the same place.